doc/source/getting_started/versioning.rst

```diff
@@ -172,4 +172,4 @@ MAPDL-supported operative systems
 You can obtain the list of MAPDL-supported operative systems on the
 `Platform Support <ansys_platform_support_>`_ page of the Ansys website.
 
-Or, you can download :ref:`ansys_current_supported_os` the list for the current release.
\ No newline at end of file
+Or, you can download the list for the current release `from here <ansys_current_supported_os_>`_. 
```

Comment:
Google developer documentation style guide says to never use "here" as a link. You want the user to know what he is going to see or download before clicking. I'd suggest making "download" the link and rewriting as "Or, you can download the list for the current release.